dataland-frontend/src/components/forms/CreateEUTaxonomyForFinancials.vue

```diff
@@ -140,30 +342,163 @@ export default defineComponent({
     };
   },
   name: "CreateEUTaxonomyForFinancials",
-  components: { DataPointFormElement, FailedUpload, FormKit, SuccessUpload, Card },
+  components: {
+    JumpLinksSection,
+    FailedUpload,
+    FormKit,
+    SuccessUpload,
+    UploadFormHeader,
+    Card,
+
+    UploadReports,
+    BasicInformationFields,
+
+    PrimeButton,
+    Calendar,
+    MultiSelect,
+    KPIfieldSet,
+  },
 
-  data: () => ({
-    innerClass: {
-      "formkit-inner": false,
-      "p-inputwrapper": true,
+  data() {
+    return {
+      formInputsModel: {} as CompanyAssociatedDataEuTaxonomyDataForFinancials,
+      filesToUpload: [] as ExtendedFile[],
+      uploadFiles: [] as ExtendedCompanyReport[],
+      fiscalYearEnd: "" as Date | "",
+      convertedFiscalYearEnd: "",
+      reportingPeriod: new Date(),
+      assuranceData: [
+        { label: "None", value: "None" },
+        { label: "LimitedAssurance", value: "LimitedAssurance" },
+        { label: "ReasonableAssurance", value: "ReasonableAssurance" },
+      ],
+      maxFileSize: UPLOAD_MAX_FILE_SIZE_IN_BYTES,
```

Comment:
Why doing this detour and not use `UPLOAD_MAX_FILE_SIZE_IN_BYTES` directly as Constant wherever it is needed?
That way a read would also directly see that this value is passed by a constant, and is not somehow calculated in a method or passed as prop.

Reply:
ok